src/ingest_validation_tools/table-schemas/antibodies.yaml

```diff
@@ -32,6 +33,18 @@ fields:
       required: False
       pattern: '1/\d+'
     example: 1/200
+  -
+    name: concentration_value
+    description: TODO!
+    type: number
+    constraints:
+      required: False
+  -
+    name: concentration_unit
+    description: TODO!
```

Comment:
```suggestion
    description: The concentration units of the antibody preparation.
```

Reply:
@mccalluc  concentration units should be optional, as the value is.

Reply:
> @mccalluc concentration units should be optional, as the value is.

Sorry, I missed this comment. I don't believe we should do it that way: We do not have a mechanism to make one field required only if another is present. Between having a default unit in the column, when there's no value for it to apply to, or missing the unit when it is needed, I prefer the former.